homeassistant/components/poolstation/entity.py

```diff
@@ -0,0 +1,37 @@
+"""Base class for Poolstation entity."""
+from __future__ import annotations
+
+from pypoolstation import Pool
+
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import PoolstationDataUpdateCoordinator
+from .const import DOMAIN
+
+
+class PoolEntity(CoordinatorEntity):
+    """Representation of a pool entity."""
+
+    coordinator: PoolstationDataUpdateCoordinator
+
+    def __init__(
+        self,
+        pool: Pool,
+        coordinator: PoolstationDataUpdateCoordinator,
+        entity_suffix: str,
+    ) -> None:
+        """Init from config, hookup pool and coordinator."""
+        super().__init__(coordinator)
+        self._pool = pool
+
+        pool_id = self._pool.id
+        name = self._pool.alias
+
+        self._attr_name = f"{name}{entity_suffix}"
+        self._attr_unique_id = f"{pool_id}{entity_suffix}"
+        self._attr_device_info = {"name": name, "identifiers": {(DOMAIN, pool_id)}}
```

Comment:
Would it make sense to have `manufacturer`, `model` etc. in the device info, too?

Reply:
It would if the payload returned that information, but I couldn't find it. I could only recognize the serial number and I wasn't sure it was that useful.